config/initializers/session_store.rb

```diff
@@ -1,4 +1,5 @@
-SERVICE_DOMAIN = (url = ENV['SERVICE_URL']) ? URI.parse(url).host : nil
+SERVICE_URL = ENV['SERVICE_URL']
+SERVICE_DOMAIN = (SERVICE_URL) ? URI.parse(SERVICE_URL).host : nil
```

Comment:
If `SERVICE_URL` isn't used anywhere else, keep it as a variable (`service_url`) instead of a constant.